credentials/oauth/oauth.go

```diff
@@ -61,6 +61,33 @@ func (ts TokenSource) GetRequestMetadata(ctx context.Context) (map[string]string
 	}, nil
 }
 
+// jwtAccess creates a JWT and send as the access token.
+type jwtAccess struct {
+	ts oauth2.TokenSource
+}
+
+func NewJwtAccessFromFile(keyFile string, audience string) (credentials.Credentials, error) {
```

Comment:
can you have a separate NewJWTAccessFromKey for users who get key from somewhere directly?